### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by Kugelsicher**

### CODE REVIEW

File: CinemaAPI/Startup.cs [hunk 1]
```diff
@@ -76,12 +75,12 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
-            if (env.IsDevelopment())
-            {
+            //if (env.IsDevelopment())
```
Comment (TrisTenuto) [inline CinemaAPI/Startup.cs:78]: Is there a reason this is just commented out? Mark will probably ask us to take it out if it's not necessary.

File: Logic/Mapper.cs [hunk 2]
```diff
@@ -1,47 +1,154 @@
...
+            {
+                movieDTO.MovieLanguages[i] = movieDTO.MovieLanguages[i].Trim();
+            }
+            
+            movieDTO.MovieTags = new List<string>();
+
+            return movieDTO;
+        }
+
+        public static MovieDTO MovieToMovieDTO(Movie movie, Rating rating, List<string> actorNames
```
Comment (TrisTenuto) [inline Logic/Mapper.cs:70]: Comments, but otherwise good

**Review by TrisTenuto — CHANGES_REQUESTED** [review]
Just a couple of minor things before I send it to Mark. Looks good!

File: CinemaAPI/Startup.cs (hunk 1 above)
Comment (Kugelsicher) [reply CinemaAPI/Startup.cs:78]: Yes, commenting that out allows swagger to work when it is deployed. Beau wanted to use swagger to test the deployment early on. It is, however, a security vulnerability; so eventually, the commented out code must be un-commented.

**Pull request closed without merging**